packages/popmotion-pose/docs/examples/react/event-focus.md

```diff
@@ -10,10 +10,10 @@ To animate an element on focus, set `focusable: true` and a `focus` prop.
 
 ```javascript
 const config = {
-  pressable: true,
+  focusable: true,
   init: { scale: 1 },
   press: { scale: 0.8 }
```

Comment:
Should be `focus`

Reply:
Thank you!
I copy code from codesandbox